scavenger-api/build.gradle.kts

```diff
@@ -50,12 +50,11 @@ dependencies {
 configure<DependencyManagementExtension> {
     imports {
         mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
```

Comment:
Maybe this is also a configuration we need??

Reply:
I think yes.
It allows us to leverage the Spring Boot BOM to simplify dependency management, ensure version consistency, and reduce the risk of version conflicts in our Spring Boot project.

> The SpringBootPlugin class provides a BOM_COORDINATES constant that can be used to import the bom without having to know its group ID, artifact ID, or version.
https://docs.spring.io/spring-boot/docs/3.2.4/gradle-plugin/reference/htmlsingle/#managing-dependencies.dependency-management-plugin.using-in-isolation